.github/workflows/claude-ci-autofix.yml

```
@@ -0,0 +1,87 @@
+        id: get_logs
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const run_id = ${{ github.event.workflow_run.id }};
+            const jobs = await github.rest.actions.listJobsForWorkflowRun({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              run_id: run_id,
+            });
+            // 'test' というジョブ名、または 'RN Unit CI' の実際のジョブ名に合わせる
+            const failedJob = jobs.data.jobs.find(job => job.conclusion === 'failure' && (job.name.includes('test') || job.name.includes('Test'))); 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)